src/strtree.c

```diff
@@ -208,25 +218,25 @@ void query_callback(void *item, void *user_data)
  * Returns the number of geometries that met the predicate or -1 in case of error.
  * */
 
-static int evaluate_predicate(FuncGEOS_YpY_b *predicate_func,
+static char evaluate_predicate(void* context,
```

Comment:
Note: the docstring above no longer matches the function signature.  Should be 
`Returns PGERR_GEOS_EXCEPTION if an error was encountered or PGERR_SUCCESS otherwise`